meetings.qmd

```diff
@@ -4,46 +4,51 @@ Please update this semester's meeting schedule [here](https://docs.google.com/sp
 
-## Logistics
+### Primers
+-   Technical primers can be great practice on teaching a topic by sharing how-to or tutorial lectures on subjects of your choosing. These are great opportunities to share a skill or approach with the group so we can all benefit from it. 
 
--   GitHub issues/project for detailed project to-do's
--   Shared Outlook Calendar for meetings, deadlines, RLEL & other events!
--   Semester planning meeting (once per sem, during the first month)
--   Slack for everything else! 😃
+-   Additionally, these are spaces where we can invite an outside presenter to share with us (for example, the [Department of Biomedical Informatics' Software Engineering Team](https://cu-dbmi.github.io/set-website/about), or the [CU Office of Diversity, Equity, Inclusion & Community Engagement](https://www.cuanschutz.edu/offices/diversity-equity-inclusion-community), or a scientist with subject expertise from another lab group). It is up to the group collectively to decide which primer topics are appropriate.
 
-### Weekly Group Meetings
+### Co-work sessions
+-   Organized as desired, and can be in-person or virtual. Have a paper or a grant deadline? Need to do some code review? Looking to catch up on recent literature? A co-work session is a great place to get these done in a focused but still social setting. These typically work best if you bring a task or to-do list, but are also good to set up regularly if you're working remotely for any extended periods of time to keep in touch and allow more natural interactions with your colleagues.  
 
-Fri: 10.30a MST
+### Weekly software engg meetings
+-   Meetings with the [DBMI SET](https://cu-dbmi.github.io/set-website/about) on software and UI/UX design and implementation, as well as work on our JRaviLab server, software applications like [MolEvolvR](https://jravilab.cuanschutz.edu/molevolvr/), or work on the [CU Alpine HPC cluster](https://www.colorado.edu/rc/alpine).
 
-We usually go around the table and give quick 5min progress updates from the week before. Alternatively, we can schedule research updates/journal club presentations each month or host separate monthly journal clubs.
+### (Bi)weekly/monthly collab meetings
+-   Less frequent except during sprints, these are meetings with our collaborators on joint projects. They may involve sharing goals, blockers, and milestones, or might involve a collaborator working as a consultant to give advice or feedback from an outside perspective on a project.    
 
-[Bioinfo primer series](https://github.com/JRaviLab/bioinfo-primers) <br> Thu: 11a--12p MST
+## Logistics
 
-### Monthly Group Socials ☕ 🍵 🥐
```

Comment:
too bad we don't do these anymore. we should absorb it into the group meeting schedule and include the occasional Lost Coffees!